digdag-standards/src/main/java/io/digdag/standards/operator/ForEachOperatorFactory.java

```diff
@@ -27,10 +27,13 @@
         implements OperatorFactory
 {
     private static Logger logger = LoggerFactory.getLogger(ForEachOperatorFactory.class);
+    private Limits limits;
 
     @Inject
-    public ForEachOperatorFactory()
-    { }
+    public ForEachOperatorFactory(Limits limits)
```

Comment:
I think this doesn't always work in this case:

* this agent runs on a remote machine.
* this agent doesn't have limits.max-workflow-tasks config (uses default value).
* server overwrites limits.max-workflow-tasks to a large number.
* a for_each> operator runs on the remote agent.

Number of tasks generated by this `for_each` operator may exceed default limits.max-workflow-tasks but it may be allowed on the server's environment.